app.yaml

```diff
@@ -27,7 +27,7 @@ handlers:
   secure: always
   redirect_http_response_code: 301
   http_headers:
-    Access-Control-Allow-Origin: http://PROJECT_ID.ew.r.appspot.com/
+    Access-Control-Allow-Origin: https://PROJECT_ID.googleplex.com/
```

Comment:
Is this ok to have googleplex.com here as external users will be using appspot?

Reply:
Good spot.